npf/types/notebook/notebook.py

```diff
@@ -8,31 +9,53 @@
 import os
 import time
 
+import pandas as pd
+from npf.graph_choice import decide_graph_type
+
 INDENT_DATA = True
 
 
-def prepare_notebook_export(datasets, all_results_df, path):
+def prepare_notebook_export(datasets: List[tuple], all_results_df:pd.DataFrame, path:str, config):
     # SIMTODO: (help) why could there be multiple datasets?
+    # TODO: with npf-compare there might be multiple dataset. Try the netperf vs iperf experiment from the examples
     dataset = datasets[0]
     test, build, runs = dataset
-    var_names = dict(datasets[0][0].config["var_names"])
-
+    var_names = dict(test.config["var_names"])
     x_vars = list(test.variables.dynamics().keys())
-    y_vars = list(list(runs.values())[0].keys())
+
+    y_vars = list(filter(lambda x:x.startswith("y_"),all_results_df.columns))
 
     # variables that get replaced in the template notebook
     variables = {
         "name": test.get_title(),
         "x_vars": x_vars,
         "x_names": get_name(x_vars, var_names),
         "y_vars": y_vars,
-        "y_names": get_name(y_vars, var_names),
+        "y_names": get_name([y[2:] for y in y_vars], var_names),
```

Comment:
@Simcornelis this could be unclear but as we have names of features like y_THROUGHPUT, it gives back THROUGPUT, LATENCY, etc.
So var_names works again.